core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -151,7 +156,7 @@ public void setNewDeviceRequest(NewDeviceRequest newDeviceRequest)
 		this.newDeviceRequest = newDeviceRequest;
 	}
 
-	private UserPrivate getUserPrivate()
+	public UserPrivate getUserPrivate()
```

Comment:
Used in:
- UserDTO for updating the User (with the nested UserPrivateDTO)
- UserService for retrieving the user as a whole
I think these are required and OK. Let's discuss if you disagree.